packages/vhd-lib/src/Vhd/VhdDirectory.js

```diff
@@ -0,0 +1,186 @@
+import { BLOCK_UNUSED } from '../../dist/_constants'
+import { createLogger } from '@xen-orchestra/log'
+import { fuFooter, fuHeader, checksumStruct } from '../_structs'
+import { SECTOR_SIZE } from '../_constants'
+import { buildHeader, buildFooter, computeBatSize } from './_utils'
+import { test, set as setBitmap } from '../_bitmap'
+import { VhdAbstract } from './VhdAbstract'
+import assert from 'assert'
+
+const { debug } = createLogger('vhd-lib:VhdDirectory')
+
+export class VhdDirectory extends VhdAbstract {
```

Comment:
Add a comment to explain the structure of such a VHD.

Reply:
I think it should be in the remoteadapter or one of the backup file, not in the VHD file